staff/lluis-bagur/app-template/home.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Home</title>
+</head>
+<body>
+
+<header>
+<h1>Home</h1>
+<nav>
+    <ul>
+        <li> <a href="register.html">Register</a></li>
+        <li> <a href="login.html">Login</a></li>
+        <li> <a href="home.html">Home</a></li>
+        <li> <a href="news.html">News</a></li>
+        <li> <a href="about.html">About</a></li>
+        <li> <a href="contact.html">Contact</a></li>
+        <li> <a href="profile.html">Profile</a></li>
+    </ul>
+</nav>
+
+</header>
+
+<main>
+        <article>
+            <header>
+                    <h2>SKYLAB COODERS ACADEMY</h2>
+            </header>
+            <section>
+                    <p> We offer coding courses where you learn web and mobile applications development.<br />
+                        We help you to learn Github + HTML + CSS + Javascript + TDD + ReactJS + NodeJS + MongoDB to find a job.</p>
+                    <br>
+                    <br>
+                    <p>In Skylab we do not just teach you programming, we make you programmer</p>
+                    <table>
```

Comment:
you seem to be using table element as a layout solution, but not as the correct purpose of this element, which is just for listing data in table format. please, if that's the case, correct it. and RTFM https://www.w3schools.com/html/html_tables.asp

Reply:
you're right, I did it by design and I forgot to undo it.